internal/jujuapi/api.go

```diff
@@ -36,9 +36,17 @@ func APIHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 // ModelHandler creates an http.Handler for "/model" endpoints.
 func ModelHandler(ctx context.Context, jimm *jimm.JIMM, p Params) http.Handler {
 	mux := http.NewServeMux()
-	mux.Handle("/", &jimmhttp.WSHandler{
+	mux.Handle("/{uuid}/api", &jimmhttp.WSHandler{
 		Upgrader: websocketUpgrader,
-		Server:   modelProxyServer{jimm: jimm},
+		Server: &apiProxier{apiServer: apiServer{
+			jimm: jimm,
+		}},
+	})
+	mux.Handle("/{uuid}/", &jimmhttp.WSHandler{
```

Comment:
there are other endpoints served by the juju contoller at /{uuid}/...

Reply:
True, I've changed this to be explicit about the endpoint.